qa/tasks/vstart_runner.py

```diff
@@ -434,8 +428,39 @@ def _do_run(self, args, check_status=True, wait=True, stdout=None,
             if os.path.exists(local_bin):
                 args = [local_bin] + args[1:]
 
+        # We have to use shell=True if any run.Raw was present, e.g. &&
+        if not shell:
+            shell = any([a for a in args if isinstance(a, Raw)])
+
         log.debug('> ' +
-                 ' '.join([str(a.value) if isinstance(a, Raw) else a for a in args]))
+                  ' '.join([str(a.value) if isinstance(a, Raw) else a for a in args]))
+
+        return args
+
+    def _perform_checks(self, args, omit_sudo, shell):
+        if isinstance(args, str):
```

Comment:
Convert list to string so we don't need to duplicate the checks.

Reply:
Done.